mmdet3d/structures/det3d_data_sample.py

```diff
@@ -162,73 +164,88 @@ def gt_instances_3d(self) -> InstanceData:
         return self._gt_instances_3d
 
     @gt_instances_3d.setter
-    def gt_instances_3d(self, value: InstanceData):
+    def gt_instances_3d(self, value: InstanceData) -> None:
```

Comment:
It would be more appropriate to indicate it as InstanceData rather than None in the typehint

Reply:
In my opinion, this function is just used to `set_field` without returning any values. It may better be `None` instead of `InstanceData`.

Reply:
ops I missed the issue. thanks for the reminder.